pages/instructorQuestionManualGrading/instructorQuestionManualGrading.js

```diff
@@ -89,7 +90,7 @@ router.get('/', (req, res, next) => {
 router.post('/', function (req, res, next) {
   if (req.body.__action === 'add_manual_grade') {
     const note = req.body.submission_note;
-    const score = req.body.submission_score_percent;
+    const score_perc = req.body.submission_score_percent;
```

Comment:
Can we change the `req.body` variable to also be `req.body.score_perc` for consistency? (not needed immediately for this PR)